changelog/install-dmd-dc.dd

```diff
@@ -0,0 +1,27 @@
+`~/dlang/install.sh` now supports `--dmd` and `--dc`
+
+`--dmd` and `--dc` has been added as a new option to the install script.
+These flags work analogous to the existing `--activate` flag and
+print information about the installed compiler binary:
+
+$(UL
+    $(LI `--dmd` returns the path of a specific compiler executable (DMD-like interface))
+    $(LI `--dc` returns the path of a specific compiler executable (custom compiler interface))
+)
+
+$(H4 Examples)
+
+$(CONSOLE curl https://dlang.org/install.sh | bash -s --dmd
```

Comment:
Not sure it's really helpful to put in this `curl` stuff.  It distracts from the example of how to use the `install.sh` script itself.

Reply:
It's how you get the installer script and would use it for a CI script.
It also provides a seamless experience for readers who want to try out the new flags as they can just copy/paste the command.

Reply:
Sure, but it would be clearer if the example was annotated to that effect, e.g. `download and run install script, printing out the installed compiler`.